backend/src/attraction/services/attraction_service.py

```diff
@@ -9,7 +9,7 @@
 from src.attraction.schemas import AttractionFilters, AttractionImages, AttractionSchema
 from src.attraction.services.geocoding_service import MapsCoService
 from src.db.cache_storage import CacheKeys, CacheStorage
-from src.db.cloudstorage import BucketNames, CloudStorage
+from src.db.cloud_storage import BucketNames, CloudStorage
 
 logger = logging.getLogger(__name__)
 cache = CacheStorage()
```

Comment:
This gonna be problematic. I would inject it through constructor as dependency injection